<launch>
    <param name="use_sim_time" value="false" />
    <param name="num_accumulated_clouds" value="1800" />
    <param name="inter_cloud_diff_" value="900" />
    

    

    <node name="cloud_to_spi" pkg="cloud_to_spi" type="node"  output="screen">
    </node>

</launch>

<!---->
<!-- <node pkg="rosbag" type="play" name="player" output="screen" args="/media/li/lavie/dataset/20201023114733_Paul_Locate.bag">
    <remap from="velodyne_points" to="velodyne_points"/>
</node> -->